e2e/commands/import.e2e.js

```diff
@@ -1810,7 +1810,64 @@ describe('bit import', function () {
       });
     });
   });
-
+  describe('import component with dependencies with yarn workspaces', () => {
```

Comment:
add the following cases:

- [ ] running it again and make sure the is no duplication in the workspaces
- [ ] add some custom field to the package.json and make sure we didn't delete it on the rewrite
- [ ] import using -p and make sure the directory has been added to the workspaces
- [ ] if manage workspaces is true but packageManager is npm / use workspaces is false, nothing happen
- [ ] if the user put some workspaces himself, they shouldn't be overriden